charts/prometheus-blackbox-exporter/README.md

```diff
@@ -62,10 +62,14 @@ securityContext:
     add: ["NET_RAW"]
 ```
 
+### To 7.0.0
```

Comment:
can you remove thats please?
7.0.0 changes already exist above.

Reply:
Yup, should be fixed.